patches/minecraft/net/minecraft/block/WoodType.java.patch

```diff
@@ -0,0 +1,39 @@
+--- a/net/minecraft/block/WoodType.java
++++ b/net/minecraft/block/WoodType.java
+@@ -18,6 +18,7 @@
+ 
+    protected WoodType(String p_i225775_1_) {
+       this.field_227045_h_ = p_i225775_1_;
++      this.namespace = "minecraft";
+    }
+ 
+    private static WoodType func_227047_a_(WoodType p_227047_0_) {
+@@ -32,4 +33,28 @@
+    public String func_227048_b_() {
+       return this.field_227045_h_;
+    }
++
++   private static final Set<WoodType> MOD_VALUES = new io.netty.util.internal.ConcurrentSet<>();
```

Comment:
Don't use netty internals, use `java.util.Collections.newSetFromMap(new java.util.WeakHashMap<>();` like we do in other places

Reply:
Ok, done.

Reply:
Eh sorry, I meant `ConcurrentHashMap`, not `WeakHashMap` here. My bad

Reply:
Fixed. Yeah, I thought something was up regarding concurrency.